event_ticket_registration_limit/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Event Ticket Registration Limit",
+    'category': 'Events/Ticket',
+    'summary': "Restrict the maximum number of tickets that can be booked in a single registration",
+    'description': """
+    A module to add a feature to restrict the maximum number of tickets per registration
+    """,
+    'version': "1.0",
+    'author': "nees",
+    'depends': [
+        'base',
+        'event',
+        'website_event'
+    ],
```

Comment:
```suggestion
    'depends': [
        'website_event'
    ],
```
Won't the above mentioned modules installed even if I keep this much diff? :thinking:

Reply:
Yes, refactored code!